modules/virt-about-reclaiming-persistent-volumes.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-about-reclaiming-persistent-volumes_{context}"]
+
+= About reclaiming persistent volumes
+
+In past releases, you accomplished persistent volume reclamation by setting
+the Storage Class *reclaimPolicy* to *recycle*. However, Kubernetes no
+longer recommends using *recycle* for persistent volume reclamation.
```

Comment:
I'd rewrite this concept to provide more information about why you'd want to reclaim a volume and what reclaiming a volume might do for you.

Reply:
Hopefully this is an improvement:

When you reclaim a volume, you unbind the PV from a persistent volume claim (PVC) 
and delete the data in the network share. Finally, you bind the PV to a new PVC to
prepare the volume for reuse.

In past releases, you accomplished persistent volume reclamation by setting
the Storage Class `reclaimPolicy` to `recycle`. Kubernetes no
longer recommends using *recycle* for persistent volume reclamation. Instead,
use the following procedure to reclaim PVs.

Reply:
Hey @bgaydosrh - I will leave the rest of the review to @kalexand-rh but I was wondering what you mean here by "past releases". What past releases are you referring to? CNV? Or Kubernetes? Since CNV hasn't had a GA, I am wondering if this makes sense here (if you are indeed referring to CNV).

Reply:
Good one, @vikram-redhat - You're right, there is no context. I'll drop that part.

Reply:
"Past releases" is still here. @bgaydosrh, PTAL

Reply:
Changed to:

Kubernetes no longer recommends setting the Storage Class `reclaimPolicy`
to `recycle` for persistent volume reclamation.
Instead, use the following procedure to reclaim PVs.